corehq/apps/hqwebapp/uglify.py

```diff
@@ -1,5 +1,4 @@
 from __future__ import absolute_import
-from __future__ import unicode_literals
```

Comment:
This doesn't seem sustainable - shouldn't this instead us `b""` in the file where relevant?

Reply:
fair enough